mlir-tensorrt/compiler/lib/Compiler/TensorRTToExecutable/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_mlir_tensorrt_library(MLIRTensorRTCompilerTensorRTToExecutable
+    TensorRTToExecutable.cpp
+    Passes.cpp
+
+    PARTIAL_SOURCES_INTENDED
+
+    DEPENDS
+    MLIRTensorRTTensorRTToExecutableIncGen
+
+    LINK_LIBS PUBLIC
+    MLIRIR
+    MLIRTensorRTRegistration
```

Comment:
This catch-all library has been removed in a commit which hasn't been sync'd up yet. Just remove this and add in dependent libraries explicitly.